### ISSUE TITLE
Bluesky Moderation Service: Change default state of Intolerant and Rude labels

### ISSUE THREAD

**Issue by spookalucca** [bluesky-social/social-app#7030]
### Summary
The default values for the Bluesky Moderation Service labeler (@moderation.bsky.app) should be changed to reflect an improved Trust & Safety experience for minorities and members of marginalized groups on the app. Intolerance should be HIDE by default and Rude should be WARN by default. 

### User Story
As a member of a marginalized group / minority, 
I want the defaults for moderation on bluesky to HIDE intolerance, and WARN about rudeness,
So that I'm better protected from hate speech on the platform while also less in jeopardy of being silenced due to tone policing. 

### Desired Implementation
Switch **Intolerance** to **HIDE** by default for all users. Switch **Rude** to **WARN** by default for all users.

### Current Defaults (as of 12-09-2024)
![image](https://github.com/user-attachments/assets/f9213bc6-6f12-4f8f-a475-0063ef172893)

### Desired Defaults
![image](https://github.com/user-attachments/assets/95d60c73-346a-48f8-a099-3ffff65bd3a0)
Switch **Intolerance** to **HIDE** by default for all users. Switch **Rude** to **WARN** by default for all users.

### Describe Alternatives

Even if you don't set Intolerance to HIDE by default (which I don't know why you wouldn't), you should at the very least change Rude to WARN by default.

### Additional Context

Intolerance has no place on a civilized platform, that's why it should be hidden by default. 

"Rudeness" being hidden by default is a form of "tone policing" that might deserve a warning but definitely not being hidden by default. This is a tool weaponized against disaffected marginalized groups who aren't being heard in order to further silence them.

Put simply, how can Rudeness be a worse offense than Intolerance on this platform?

**Comment by spotlight2k** [comment]
Hate speech has no place anywhere

**Comment by Dingirge** [comment]
Bluesky has to start taking actual responsibility for trust & safety, instead of only managing PR around it. 

Providing community moderation tools means little to nothing when defaults facilitate the domination and silencing of marginalised groups.

This feature change would be a step in the right direction.

**Comment by Lapidearum** [comment]
Yeah, I dunno, maybe hate speech is a more serious offense than being a little mean? Please fix this.

**Comment by LemmaEOF** [comment]
This is an extremely sensible change to make, and should absolutely be the default.

**Comment by gdude2002** [comment]
This is a very sensible proposal, although honestly, I feel like "Rude" might be better defaulted to "Off" given how it's been applied to minorities and activists historically.

I dunno, it seems like an odd thing to filter in general - but it *definitely* shouldn't be considered more serious than intolerance, which is a label you'd expect to be applied to bigots like Singal.

**Comment by spookalucca** [comment]
> This is a very sensible proposal, although honestly, I feel like "Rude" might be better defaulted to "Off" given how it's been applied to minorities and activists historically.
> 
> I dunno, it seems like an odd thing to filter in general - but it _definitely_ shouldn't be considered more serious than intolerance, which is a label you'd expect to be applied to bigots like Singal.

Taking it a step further with Rude, bold! Someone must have grown up with a thick skin ;)

**Comment by SpriterGors** [comment]
I vehemently agree with this change.

Additionally, I honestly don't even like the existence of "Rude" because how do you even define what's rude? Of course, going around and cussing at people randomly IS rude, but we all know that in face of great prejudice, the affected people will take actions that normally would be perceived as "rude". Should them also be labelled rude? Paraphrasing @gdude2002 the label "Rude" has been used to label minorities and activists denouncing problems, and honestly that is not a path I personally want to see Bluesky taking.

I know that T&S is difficult and with a site with +6 million people it is a gargantuan task, so at least the tools we use to curate and protect ourselves should be working 100% correctly. I personally would focus this label on more glaring misconducts like "intolerance" and "threat", not the simple act of being rude.

**Comment by paskausks** [comment]
The proposal makes sense to me. Also, intolerance is quite easily definable, while, as mentioned before, rudeness is not.

**Comment by SnoFox** [comment]
Playing devils advocate here:
I believe “warn” is a harsher action than “hide.”

In the current setup, rude posts are effectively deboosted; it simply disappears a post from most people’s feeds.

On the other hand, warning on a post actively lets users know what sort of behavior to expect from the account, providing an opportunity to block.

Personally I was surprised to find that rude was hide by default but using this line of thinking I actually understand why it may have launched as-is. Everyone can have an off day and might post something rude. I’d prefer that not to damper my mood, so hide it. However, if someone is being intolerant, awareness is the solution, not limiting their reach to just their echo chamber.

**Comment by gdude2002** [comment]
While I see what you're getting at @SnoFox, I feel like that's why we're able to configure the labels. From a platform perspective, I feel it makes more sense to hide intolerance to pre-emptively limit the reach - and thus the motivation - of that form of problematic content.

In my experience, many accounts that appear to be suitable for this label use intolerance as a vehicle for exposure and agenda. In my opinion, it's worth hiding ahead of time - and, assuming accounts are tagged properly, this means many people will never see the problematic content and thus won't need to block the accounts!

That's the way I see it, at least.

**Comment by SpriterGors** [comment]
I can understand this line of thought, but a hypothetical situation is: user A, a known bad actor, posts intolerant stuff under the guise of "just asking questions" constantly; user B loses their cool with the constant harassment and insults A; B's post is now labeled as "Rude" and hidden from the feed, while user A can continue posting incendiary things under the guise of "just asking questions".

I have seen many times that bad actors will teeter on the edge of what is considered permitted and what isn't in order to bait people to insult them. While we know that the ideal is to never deal with trolls and bad actors, we know this is not usually what happens. I think it is better to deplatform bad actors by hiding them altogether, since they're not strangers to gaming the system in this way

**Comment by spookalucca** [comment]
> @SnoFox I can understand this line of thought, but a hypothetical situation is: user A, a known bad actor, posts intolerant stuff under the guise of "just asking questions" constantly; user B loses their cool with the constant harassment and insults A; B's post is now labeled as "Rude" and hidden from the feed, while user A can continue posting incendiary things under the guise of "just asking questions".
> 
> I have seen many times that bad actors will teeter on the edge of what is considered permitted and what isn't in order to bait people to insult them. While we know that the ideal is to never deal with trolls and bad actors, we know this is not usually what happens. I think it is better to deplatform bad actors by hiding them altogether, since they're not strangers to gaming the system in this way

Just want to +1 the fact that bad actors will "goad" other people with intolerant speech to bait a "rude" or angry response from a marginalized member, who then gets punished more than the goading bad actor. This is something that plays out a lot on social media, although I first learned of this tactic's potential in The Elders Scrolls III: Morrowind.

**Comment by Aquilux** [comment]
The defaults of a website not just signal the mentality of the people behind it, it also has a major impact on the general user-base as many if not a majority of users will leave the settings untouched. In this case leaving the defaults as they are both signals that tone is more important than whether or not something is hate speech, and leaves users more exposed to hate speech (especially when wrapped in a veneer of civility), allowing such harmful ideas to become the norm over the voices of those who would find such things offensive and are willing to speak out against it.

**Comment by Aquilux** [comment]
It's important to point out, @SnoFox , that people who are trying to spread hate and intolerance could care less about a slap on the wrist and being called out. They're expecting that. They're ready for that. They don't care. The only thing that putting a warning on their content does as opposed to just making it disappear for most people is that it gives them more access to the user-base and more vulnerable targets (whether it be targets of hate or targets for conversion).

**Comment by KingMob4313** [comment]
Hate speech and bigotry is unacceptable on civilized society and is even plain-law illegal in some regions.

**Comment by GodOfKnockers** [comment]
This is a very practical request. I say make it happen.

**Comment by frackham** [comment]
Poppers paradox of intolerance.  100% agree.

**Comment by pedrosaurus** [comment]
This makes total sense, hope they implement the proposed changes.

**Comment by Dingirge** [comment]
This should be prioritized and adopted as soon as possible.

**Comment by pixelbuffer-design** [comment]
I fully support this proposed change.

**Comment by InkstainTheBat** [comment]
Agreed, this change needs to be brought!

**Comment by xtien** [comment]
The moderation I see on Bluesky is censorship. I did not switch from Twitter to Bluesky to get censorship again.

**Comment by ygg2** [comment]
Noting here: it's been described that the "Rude" label is applied to replies from people who aren't close in the poster's social graph (as in haven't interacted with the person or their friends) that hit a toxicity filter, intended to catch drive-by harassment.

I saw some suggest it be renamed for accuracy and I think that should be considered.

I don't know if anything's been said about top level or user-reported "Rude" labels, but they should probably be split off to another category since they have different criteria.

**Comment by rcgit989** [comment]
All I can say is that these new rules are so heavy handed its just a way for Liberal Karens to shadow ban people they do not like. I mean are you serious that it is a violation of the site to reply to a post that didnt specifically ask for replies? Really? Are you guys 5 year olds? Why cant you block people on your own without mommy and daddy creating a bubble for you and all your baby friends? You guys really honestly and truly are why we have Trump. Liberals cant even disagree with other Liberals anymore. You should be listening to people who agree with you on 80-90% of everything. I simply cannot believe how weak an intolerant Liberals are. A core value of being a Liberal USED to be tolerance. And do not even dare hide behind racism you guys simply cannot deal with any human being who makes you think or feel anything you do not want to and that makes you really weak.

**Comment by rcgit989** [comment]
> Playing devils advocate here: I believe “warn” is a harsher action than “hide.”
> 
> In the current setup, rude posts are effectively deboosted; it simply disappears a post from most people’s feeds.
> 
> On the other hand, warning on a post actively lets users know what sort of behavior to expect from the account, providing an opportunity to block.
> 
> Personally I was surprised to find that rude was hide by default but using this line of thinking I actually understand why it may have launched as-is. Everyone can have an off day and might post something rude. I’d prefer that not to damper my mood, so hide it. However, if someone is being intolerant, awareness is the solution, not limiting their reach to just their echo chamber.
> 
> If users are uninterested in helping in combating intolerant people, they can adjust the settings, provided they were a bit less hidden by default.

the rude label is a very obvious and very lazy way of shadow banning people the moderators themselves do not like or agree with and honestly if you run a social media site like that you are creating a bubble that will ruin your site. People sitting around agreeing with one another on things and banning anyone they do not agree with is creating a lot more harm than good. No wonder no message coming from bluesky has any value outside of it.

**Comment by Stormlondon** [comment]
> All I can say is that these new rules are so heavy handed its just a way for Liberal Karens to shadow ban people they do not like. I mean are you serious that it is a violation of the site to reply to a post that didnt specifically ask for replies? Really? Are you guys 5 year olds? Why cant you block people on your own without mommy and daddy creating a bubble for you and all your baby friends? You guys really honestly and truly are why we have Trump. Liberals cant even disagree with other Liberals anymore. You should be listening to people who agree with you on 80-90% of everything. I simply cannot believe how weak an intolerant Liberals are. A core value of being a Liberal USED to be tolerance. And do not even dare hide behind racism you guys simply cannot deal with any human being who makes you think or feel anything you do not want to and that makes you really weak.

You are close. In practice(from what I've observed personally) it's a way for Zionist Karens to shadow ban people speaking out again UN recognized GENOCIDE.
